examples/make_assets/ptable.py

```diff
@@ -86,9 +87,7 @@
 
 # %% Histograms laid out in as a periodic table
 # Generate random parity data with y \approx x with some noise
-data_dict = {
-    elem.symbol: np.random.randn(100) + np.random.randn(100) for elem in Element
-}
+data_dict = {elem.symbol: np_rng.randn(100) + np_rng.randn(100) for elem in Element}
```

Comment:
@janosh It's so interesting, as I started working on fixing `NPY002` violations in `pymatgen` https://github.com/materialsproject/pymatgen/pull/3892 without looking at this PR almost at the same time (really!).

But this legacy to new Generator replacement is not just replacing `np.random` with `rng` (the grammar is not exactly the same, please refer to https://numpy.org/doc/stable/reference/random/generator.html), this should be `np_rng.standard_normal(100)` instead.

Reply:
good catch! thanks for letting me know. i should really take the time to make these scripts run in CI to avoid such mistakes...

will submit a fix now

Reply:
No problem at all :) Yes we need to include them into CI tests, mind letting me (learn and) do this?